### DISCUSSION TITLE
Change an account's profile picture

### DISCUSSION THREAD

**Discussion by mycroftcanner** [sbertix/Swiftagram#151] [Ideas]
Is it possible to change an account's profile picture using Swiftagram? I quickly look through the repo and couldn't find anything.

**Comment by sbertix** [comment]
Hey @mycroftcanner, 

I've moved this to the **Discussions** section, as this is not a bug-related issue. 

You're right, this is not yet supported in the library.  
I might add this in time for `5.0.0`, though.
